### ISSUE TITLE
service doesn't terminate on sigterm

### ISSUE THREAD

**Issue by matti** [kontena/akrobateo#26]
```
service-lb-postgres-akrobateo-8s7bs   0/1     Terminating   0          38s
```

I think it ignores SIGTERM signal

**Comment by matti** [comment]
it's especially annoying because it's created by the operator and I can not force `terminationGracePeriodSeconds: ` as with other misbehaving images

**Comment by matti** [comment]
I think this ignores TERM https://github.com/kontena/akrobateo/blob/master/lb-image/entrypoint.sh#L24
here's what I always do:
```
tail -f /dev/null &
wait $!
```

**Comment by jnummelin** [comment]
fixed in #28
